content/docs/studio/selfhosted/configuration/index.md

```diff
@@ -0,0 +1,76 @@
+# Configuration
+
+Studio has a unified YAML configuration file, often referred to as `values.yaml`
+in the documentation.
+
+## Updating the configuration
+
+To update the configuration and reload Studio, follow the instructions according
+to your installation method:
+
+<toggle>
```

Comment:
I wouldn't do. a toggle here. We're using helm commands so configuration flow is unified, I would only add an admon/note that you have to run the command from a machine with kubectl access to the cluster, so if you're using an AMI (specifically), you may have to SSH into the instance.

no need tor tabs then